src/containers/Suggestions/Suggestions.js

```diff
@@ -1,120 +1,112 @@
 import React, { useEffect, useState } from "react";
+import { withRouter } from "react-router-dom";
 import { getSuggestions } from "../../services/suggestions";
 import axios from "axios";
+import { buildBundle } from "../../services/backendCalls";
 import "./Suggestions.css";
 
-export default props => {
-    const { duration } = props;
+export default withRouter(props => {
+  const { destination, duration, departureDate, returnDate } = props;
 
-    const [categories, setCategories] = useState(null);
-    const [items, setItems] = useState(null);
-    const [displayItems, setDisplayItems] = useState(null);
-    const [currCategory, setCurrCategory] = useState(null);
+  const [categories, setCategories] = useState(null);
+  const [items, setItems] = useState(null);
+  const [displayItems, setDisplayItems] = useState(null);
+  const [currCategory, setCurrCategory] = useState(null);
+  const [newDisplay, setNewDisplay] = useState(null);
 
-    useEffect(() => {
-        const allTheItems = getSuggestions(duration);
-        setItems(allTheItems);
+  useEffect(() => {
+    const allTheItems = getSuggestions(duration);
+    setItems(allTheItems);
 
-        axios({
-        method: "get",
-        url: "http://localhost:5000/categories/all"
-        })
-        .then(res => {
-            setCategories(res.data);
-        })
-        .catch(err => {
-            console.log(err);
-        });
-    }, []);
+    axios({
+      method: "get",
+      url: "http://localhost:5000/categories/all"
```

Comment:
We have in the services file, the backendUrlConnect, so you can replace the http://localhost:5000/ with the backendUrlConnect.